elrepo-libata.patch

extern void blk_queue_flush_queueable(struct request_queue *q, bool queueable);
is in include/linux/blkdev.h in RHEL 8.0. However it was removed in RHEL 8.1.

Submitted-by: Akemi Yagi <toracat@elrepo.org>

--- a/libata.h	2019-07-26 03:18:45.000000000 -0700
+++ b/libata.h	2019-12-14 16:55:47.127481325 -0800
@@ -146,6 +146,7 @@ extern int ata_scsi_user_scan(struct Scs
 			      unsigned int id, u64 lun);
 int ata_sas_allocate_tag(struct ata_port *ap);
 void ata_sas_free_tag(unsigned int tag, struct ata_port *ap);
+extern void blk_queue_flush_queueable(struct request_queue *q, bool queueable);
 
 
 /* libata-eh.c */
